src/GUI/gameloop.py

```diff
@@ -70,12 +94,154 @@ def setup(self, tmx_maps, player_start_pos):
+
+        DIFF_TEXT = self.get_font(50).render('DIFFICULTY', True, 'black')
+        DIFF_RECT = DIFF_TEXT.get_rect(center=(623, 100))
+
+        easy_button_surface = pygame.Surface((200, 100))
+        easy_button_surface.fill('white')
+        medium_button_surface = pygame.Surface((200, 100))
+        medium_button_surface.fill('white')
+        hard_button_surface = pygame.Surface((200, 100))
+        hard_button_surface.fill('white')
+        back_button_surface = pygame.Surface((100, 100))
+        back_button_surface.fill('white')
+
+        EASY_BUTTON = Button(image=easy_button_surface, pos=(623, 250), text_input="Easy", font=self.get_font(35), base_color='grey', hovering_color='black')
+        MEDIUM_BUTTON = Button(image=medium_button_surface, pos=(623, 365), text_input="Medium", font=self.get_font(35), base_color='grey', hovering_color='black')
+        HARD_BUTTON = Button(image=hard_button_surface, pos=(623, 475), text_input="Hard", font=self.get_font(35), base_color='grey', hovering_color='black')
+        BACK_BUTTON = Button(image=back_button_surface, pos=(100, 650), text_input="< Back", font=self.get_font(40), base_color='grey', hovering_color='black')
+
+        for button in [EASY_BUTTON, MEDIUM_BUTTON, HARD_BUTTON, BACK_BUTTON]:
+            button.changeColor(DIFF_MOUSE_POS)
+            button.update(self.screen)
+
+        self.screen.blit(DIFF_TEXT, DIFF_RECT)
+
+        return BACK_BUTTON
+
+    def main_menu(self):
+        self.screen.fill('black')
+
+        MENU_MOUSE_POS = pygame.mouse.get_pos()
+
+        MENU_TEXT = self.get_font(100).render('PYSEAS', True, 'white')
+        MENU_RECT = MENU_TEXT.get_rect(center=(620, 100))
+
+        # play_button_surface = pygame.Surface((300, 100))
+        # play_button_surface.fill("brown")
+
+        play_button_surface = pygame.Surface((370, 100))
+        # play_button_surface.fill("brown")
+        opt_button_surface = pygame.Surface((370, 100))
+        # opt_button_surface.fill("brown")
+        quit_button_surface = pygame.Surface((220, 100))
+        # quit_button_surface.fill("brown")
+
+        PLAY_BUTTON = Button(image=play_button_surface, 
+                                pos=(623, 250), text_input='PLAY', font=self.get_font(45), base_color='white', hovering_color='grey')
+        
+        OPT_BUTTON = Button(image=opt_button_surface, 
+                            pos=(623, 360), text_input='OPTIONS', font=self.get_font(45), base_color='white', hovering_color='grey')
+        
+        QUIT_BUTTON = Button(image=quit_button_surface, 
+                            pos=(623, 470), text_input='QUIT', font=self.get_font(45), base_color='white', hovering_color='grey')
+        
+        self.screen.blit(MENU_TEXT, MENU_RECT)
+
+        for button in [PLAY_BUTTON, OPT_BUTTON, QUIT_BUTTON]:
+            button.changeColor(MENU_MOUSE_POS)
+            button.update(self.screen)
+
+        return PLAY_BUTTON, OPT_BUTTON, QUIT_BUTTON
```

Comment:
I think this could be a problem with later versions if you expect more than one returns